test/sandbox_test.py

```diff
@@ -152,6 +152,32 @@ def test_sandbox_stdin_write_after_eof(app, servicer):
         sb.stdin.write(b"foo")
 
 
+@skip_non_linux
+def test_sandbox_stdout(app, servicer):
```

Comment:
I think this way of testing is a good idea, but I'm not actually convinced that the logs look how you expect. Can you add an `io_streams_test.py` file instead and use the servicer to manually override `SandboxGetLogs` to ensure that the log messages actually get aggregated or split properly? Here's an example of how this works for app logs: 
https://github.com/modal-labs/modal-client/blob/pawalt/parampoint/test/app_test.py#L366

Reply:
Nice yeah you can keep these, but add a comment describing what you're doing.

Reply:
What do we expect the logs to look like? I think the `echo` comes with an extra newline which is why it looks weird. Added `io_streams_test` to test the aggregating more explicitly.